tasks/steps/failover_promote.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- tags:
+    - cartridge-config
+    - molecule-idempotence-notest
```

Comment:
It's not good that you are using special tag to skip it on idempotence test. Let's take a step idempotency!